src/main/java/com/github/gilday/darkmode/DarkModeSync.java

```diff
@@ -26,16 +29,27 @@
  */
 public final class DarkModeSync implements Disposable {
 
-  private final ScheduledFuture<?> scheduledFuture;
+  private ScheduledFuture<?> scheduledFuture;
 
-  private final LafManager lafManager;
-  private final DarkModeSyncThemes themes;
-  private final Alarm updateOnUIThreadAlarm;
+  private LafManager lafManager;
+  private DarkModeSyncThemes themes;
+  private Alarm updateOnUIThreadAlarm;
 
-  /** @param lafManager IDEA look-and-feel manager for getting and setting the current theme */
-  public DarkModeSync(final LafManager lafManager) {
+  private static final class MyStartupActivity implements StartupActivity.DumbAware {
```

Comment:
Can we make this listener `public`? I assume `private` works because the platform is using `setAccessible` before using reflection to create a new instance of this listener, but I would rather not rely on that behavior.

Reply:
Done